### ISSUE TITLE
blankToVariable applies to INSERT/DELETE data

### ISSUE THREAD

**Issue by rubensworks** [joachimvh/SPARQLAlgebra.js#74]
When the `blankToVariable` option is enabled, blank nodes that occur in INSERT/DELETE (DATA) are also transformed into variables, which is not expected (and even illegal according to the spec).

Example:
```javascript
const {translate, toSparql, toSparqlJs} = require('sparqlalgebrajs');
const query = 'INSERT DATA { GRAPH <ex:g1>  { _:b <ex:p> <ex:o> } } ;';
const algebra = translate(query, { quads: true, blankToVariable: true });
console.log(JSON.stringify(algebra, null, '  '));
```

Output:
```json
{
  "type": "deleteinsert",
  "insert": [
    {
      "termType": "Quad",
      "value": "",
      "subject": {
        "termType": "Variable",
        "value": "e_b"
      },
      "predicate": {
        "termType": "NamedNode",
        "value": "ex:p"
      },
      "object": {
        "termType": "NamedNode",
        "value": "ex:o"
      },
      "graph": {
        "termType": "NamedNode",
        "value": "ex:g1"
      },
      "type": "pattern"
    }
  ]
}
```

**Comment by joachimvh** [comment]
Potentially this can be caught early after which the boolean can just be reset. It surprises me actually that you can use blank nodes where variables are forbidden in update queries.

**Comment by rubensworks** [comment]
Closing this, as this is in fact something that should be caught in SPARQL.js, as it's (according to the spec) a syntactical error.

**Comment by rubensworks** [comment]
This seems to be an issue after all.
Bnodes are disallowed in DELETE DATA/DELETE WHERE queries, but they _are_ allowed in INSERT queries,
in which case the bnodes should not be transformed into variables.

**Comment by joachimvh** [comment]
Fixed in f62a8b5efc948df66898b3a369ec35f059ab7a70
